src/utils/canvas.ts

```diff
@@ -240,8 +254,21 @@ class CanvasCard {
       return Promise.reject("no-image");
     }
 
+    this.furniture = furniture;
+
+    if (this.drawCount > 0) {
```

Comment:
Would this be better expressed as a boolean, or are there situations where it's OK for `drawCount > 1`?

Reply:
Good point, I just changed how it worked slightly and a boolean is now a better fit.

Reply:
I've updated this!